services/download/src/main/java/app/coronawarn/server/services/download/normalization/DecodedDsosNotInExposureNotificationFrameworkRange.java

```diff
@@ -0,0 +1,11 @@
+package app.coronawarn.server.services.download.normalization;
+
+public class DecodedDsosNotInExposureNotificationFrameworkRange extends RuntimeException {
```

Comment:
Suggested naming:
"InvalidDsosException" or something shorter than the current name which ends with '*Exception'.